### ISSUE TITLE
dub can't find any packages 

### ISSUE THREAD

**Issue by DiddiZ** [lindt/docker-ldc#4]
In the ldc image dub seems unable to retrieve any package:

```
Step 4/4 : RUN dub test
 ---> Running in f265d9ecdb5c
Root package evaluation-d references unknown package fluent-asserts
Removing intermediate container f265d9ecdb5c
The command '/bin/sh -c dub upgrade' returned a non-zero code: 2
```
I tried different dependencies with the same result. `code.dlang.org` is reachable currently, as demonstrated by using `dlanguage/dmd`.

`dub.json`:
```json
{
	"name": "evaluation-d",
	"dependencies": {
		"fluent-asserts": "~>0.7",
	}
}
```

`Dockerfile`:
```
FROM dlanguage/ldc

ADD dub.json /usr/src/rk-mir/

WORKDIR /usr/src/rk-mir/

RUN dub upgrade
```

Using `FROM dlanguage/dmd` instead works as expected:
```
Step 4/4 : RUN dub upgrade
 ---> Running in b6b467fc633b
Upgrading project in /usr/src/rk-mir
Fetching fluent-asserts 0.7.0 (getting selected version)...
Fetching libdparse 0.7.1 (getting selected version)...
Fetching ddmp 0.0.1-0.dev.2 (getting selected version)...
```

As I want to use `mir-glas` switching from `ldc` to `dmd` is no option for me.

**Comment by DiddiZ** [comment]
Solution: Installing `libcurl3` before running dub solves the issue.

`RUN apt-get update && apt-get install -y libcurl3`

Currently, the `dmd` image installs `libcurl3`, but the `ldc` image doesn't.

**Comment by lindt** [comment]
it works now.....can we close this issue?

```
$ docker run -ti dlanguage/ldc dub search fluent-asserts
==== registry at http://code.dlang.org/ ====
fluent-asserts (0.7.1)      Fluent assertions done right
fluent-logger (0.1.2)       A structured logger for Fluentd.
pyjamas (0.2.1)             A BDD assertion library for D
exceptionhandling (0.1.0)   Functions to make exception and assertion handling in D easier.
assert_that (0.0.4)         An assert function with pattern matching
chap (0.4.0)                Asserts toolkit
```

**Comment by DiddiZ** [comment]
Yes. This was meant to be closed with #6 aswell.
